### PR TITLE
[bundles/resnet50] Read from the right output tensor

### LINKED ISSUE #1604: Could not run the ResNet-50 example of creating the standalone bundles
This is my log, after running `make run`:
```
File ?resnet50/predict_net.pbtxt? already there; not retrieving.

File ?resnet50/predict_net.pb? already there; not retrieving.

File ?resnet50/init_net.pb? already there; not retrieving.

image-classifier: for the -model_input_name option: must be specified at least once!
Makefile:43: recipe for target 'build/resnet50.o' failed
make: *** [build/resnet50.o] Error 1
```

After adding `-model_input_name=gpu_0/data ` to line 44 in the make file, I get 
```
File ?resnet50/predict_net.pbtxt? already there; not retrieving.

File ?resnet50/predict_net.pb? already there; not retrieving.

File ?resnet50/init_net.pb? already there; not retrieving.

Allocated weights of size: 102121984
Expected weights of size: 102121984
Loaded weights of size: 102121984 from the file resnet50.weights
Allocated mutable weight variables of size: 606144
Loaded image: /datadrive/bhaskar/renesasai/glow/tests/images/imagenet/cat_285.png
Loaded images size in bytes is: 602112
Copying image data into mutable weight vars: 602112 bytes
Expected to find variable 'output'
resnet50: resnet50.cpp:267: const SymbolTableEntry &getMutableWeightVar(const BundleConfig &, const char *): Assertion `mutableWeightVar && "Expected to find a mutable weight variable"' failed.
Aborted (core dumped)
Allocated weights of size: 102121984
Expected weights of size: 102121984
Loaded weights of size: 102121984 from the file resnet50.weights
Allocated mutable weight variables of size: 606144
Loaded image: /datadrive/bhaskar/renesasai/glow/tests/images/imagenet/dog_207.png
Loaded images size in bytes is: 602112
Copying image data into mutable weight vars: 602112 bytes
Expected to find variable 'output'
resnet50: resnet50.cpp:267: const SymbolTableEntry &getMutableWeightVar(const BundleConfig &, const char *): Assertion `mutableWeightVar && "Expected to find a mutable weight variable"' failed.
Aborted (core dumped)
Allocated weights of size: 102121984
Expected weights of size: 102121984
Loaded weights of size: 102121984 from the file resnet50.weights
Allocated mutable weight variables of size: 606144
Loaded image: /datadrive/bhaskar/renesasai/glow/tests/images/imagenet/zebra_340.png
Loaded images size in bytes is: 602112
Copying image data into mutable weight vars: 602112 bytes
Expected to find variable 'output'
resnet50: resnet50.cpp:267: const SymbolTableEntry &getMutableWeightVar(const BundleConfig &, const char *): Assertion `mutableWeightVar && "Expected to find a mutable weight variable"' failed.
Aborted (core dumped)
Makefile:22: recipe for target 'run' failed
make: *** [run] Error 134
```

Can someone point out if I have to make any more changes to get it running?

### PR DESCRIPTION
**Pull request by qcolombet**
*Description*:
At some point the output of the network changed from "output" to
"gpu_0/softmax". Reflect that when reading the result from the
bundle.

*Testing*:
Ran the makefile of the bundle of resnet50

*Documentation*: N/A
[Optional Fixes #issue]
This fixes #1604

### CODE REVIEW

**Comment by jfix71** [comment]
I have a feeling this is due to changes I made to ProtobufLoader when I needed it to support multiple outputs in a proto, so this might be an issue for the other bundle examples too. Did you check if `vgg19` and `zfnet512` also need to have this fixed?

**Comment by qcolombet** [comment]
> Did you check if vgg19 and zfnet512 also need to have this fixed?

No, I didn't. I assumed they were working. Let me check.

**Comment by qcolombet** [comment]
We likely have a similar problem:
- vgg19 output is now: "prob"
- zfnet512 output is now: "gpu_0/softmax"

Waiting for the runs to finish and I'll push fixes for those too.

Question: Why don't we catch these breakages in our CI?

**Comment by rdzhabarov** [comment]
> Question: Why don't we catch these breakages in our CI?

Partly because we need to download all the weights and models. Similar to https://github.com/pytorch/glow/issues/1257.

**Comment by qcolombet** [comment]
Fixed the other bundles in #1610.
